store/storetest/channel_store.go

```diff
@@ -3282,28 +3074,23 @@ func testChannelStoreAnalyticsDeletedTypeCount(t *testing.T, ss store.Store) {
 	require.Nil(t, err)
 
 	d4, err := ss.Channel().CreateDirectChannel(u1, u2)
-	if err != nil {
-		t.Fatalf(err.Error())
-	}
+	require.Nil(t, err, err.Error())
```

Comment:
Why did previous build fail for this line?
I have not used any new function.

Reply:
because you are executing `err.Error()` in all cases, no matter if the `err` variable is `nil` or not. You shouldn't need to pass the `err.Error()` third parameter here, because is going to be shown anyway as part of the error message from the assertion. my suggestion is just remove the `err.Error()` third parameter there.

Reply:
Updated PR to remove third parameter